Fhi.HelseId.Samples/Authentication/FakeAuthenticationSamples.cs

```diff
@@ -0,0 +1,204 @@
+﻿using System.Net;
+using System.Security.Claims;
+using Fhi.HelseId.Common.Identity;
+using Fhi.HelseId.Integration.Tests.TestFramework;
+using Fhi.HelseId.Web;
+using Fhi.HelseId.Web.ExtensionMethods;
+using Fhi.TestFramework.AuthenticationSchemes.CookieScheme;
+using Fhi.TestFramework.Extensions;
+using Fhi.TestFramework.NHNTTT;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+
+namespace Fhi.Samples.TestFramework.Authentication
+{
+    internal class FakeAuthenticationSamples
```

Comment:
Just have to say it again, I really love these minimal API tests, such a great find!

Reply:
Thank you :)